src/pages/workspaces/workspace/Data.js

```diff
@@ -1002,6 +1025,28 @@ const WorkspaceData = _.flow(
               }
             }, [`${type} (${typeDetails.count})`])
           }, sortedEntityPairs),
+          div({ style: Style.navList.heading }, ['Snapshots']),
+          _.map(([snapshotName, { id: snapshotId, entityMetadata: snapshotTables }]) => {
+            const snapshotTablePairs = _.flow(_.toPairs, _.sortBy(_.first))(snapshotTables)
+            return h(Collapse, {
+              key: snapshotName,
+              buttonStyle: { color: colors.dark(), fontWeight: 600, marginBottom: 0 },
+              style: { fontSize: 14, lineHeight: '50px', paddingLeft: '1.5rem', borderBottom: `1px solid ${colors.dark(0.2)}` },
+              title: snapshotName
+            }, [
+              div({ style: { fontSize: 14, lineHeight: '1.5' } }, [
+                div(_.map(([tableName, { count }]) => {
```

Comment:
nit: This inner div isn't necessary, the `_.map` expression can just sit inside the children array of the outer div.